<sdf version="1.7">
  <world name="default">
    <!-- A global light source -->
    <include>
      <uri>model://sun</uri>
    </include>

    <model name="heightmap">
      <static>true</static>
      <link name="heightmap_argos">
        <collision name="collision">
          <geometry>
            <heightmap>
              <uri>model://tif_files/cropped_gdal.tif</uri>
              <size>80 80 10</size>
              <pos>0 0 -320</pos>
            </heightmap>
          </geometry>
        </collision>

        <visual name="visual_abcedf">
          <geometry>
            <heightmap>
              <texture>
                <diffuse>file://media/materials/textures/dirt_diffusespecular.png</diffuse>
                <normal>file://media/materials/textures/flat_normal.png</normal>
                <size>1</size>
              </texture>
              <texture>
                <diffuse>file://media/materials/textures/grass_diffusespecular.png</diffuse>
                <normal>file://media/materials/textures/flat_normal.png</normal>
                <size>1</size>
              </texture>
              <texture>
                <diffuse>file://media/materials/textures/fungus_diffusespecular.png</diffuse>
                <normal>file://media/materials/textures/flat_normal.png</normal>
                <size>1</size>
              </texture>
              <blend>
                <min_height>2</min_height>
                <fade_dist>5</fade_dist>
              </blend>
              <blend>
                <min_height>4</min_height>
                <fade_dist>5</fade_dist>
              </blend>
              <uri>model://tif_files/cropped_gdal.tif</uri>
              <size>80 80 10</size>
              <pos>0 0 -320</pos>
            </heightmap>
          </geometry>
        </visual>

      </link>
    </model>

    <model name='texture_1'>
      <static>1</static>
      <link name='link'>
        <visual name='visual'>
          <geometry>
            <mesh>
              <uri>model://../../mini_project/meshes/vines.stl</uri>
              <scale>0.03 0.03 0.03</scale>
            </mesh>
          </geometry>
        </visual>
        <collision name='right-angle'>
          <geometry>
            <mesh>
              <uri>model://../../mini_project/meshes/vines.stl</uri>
              <scale>0.03 0.03 0.03</scale>
            </mesh>
          </geometry>
          <max_contacts>10</max_contacts>
          <surface>
            <contact>
              <ode/>
            </contact>
            <bounce/>
            <friction>
              <torsional>
                <ode/>
              </torsional>
              <ode/>
            </friction>
          </surface>
        </collision>
        <self_collide>0</self_collide>
        <enable_wind>0</enable_wind>
        <kinematic>0</kinematic>
      </link>
      <pose>0 2 1 0 -3.13</pose>
    </model>

  </world>
</sdf>
    